### ISSUE TITLE
How should we handle when people want items taken down?

### ISSUE THREAD

**Issue by rtilla1** [lsulibraries/repository#6]
There is a fancy word for this...redacted? re...something. I think there's already a module that does this, but because I can't remember the word, I cannot find it.

**Comment by jpeak5** [comment]
you mean when, for example, a faculty member wants her dissertation to 
remain private until the book can come out ?

where does the concept of *embargo* fit ?

**Comment by rtilla1** [comment]
Yes, embargo is the word I was looking for. I think this module is the one I was thinking of: https://github.com/Islandora/islandora_ip_embargo There are also some interesting comments made about embargo on the Google group archive that may be helpful to us.
